src/app/CommandHandler.cpp

```diff
@@ -759,12 +770,21 @@ CommandHandler::Handle::Handle(CommandHandler * handle)
     }
 }
 
-CHIP_ERROR CommandHandler::Finalize(System::PacketBufferHandle & commandPacket)
+CHIP_ERROR CommandHandler::FinalizeInvokeRequestMessage(bool aHasMoreChunks)
```

Comment:
This function's name makes no sense.  CommandHandler never sends InvokeRequestMessage....

Reply:
Ops, typo should be FinalizeInvokeResponseMessage